### ISSUE TITLE
fixture 'epoch' not found

### ISSUE THREAD

**Issue by Meerkhani** [vercel/hyper#6175]
def test(epoch):
    model.eval()
    test_loss = 0
    with torch.no_grad():
        for batch_idx, (data, target) in enumerate(test_loader):
            data, target = data.to(device), target.to(device)
            output = model(data)
            loss = criterion(output, target)
            test_loss += loss.item()

            if batch_idx == 0:
                n = min(data.size(0), 4)

    test_loss /= len(test_loader.dataset)
    print('====> Test set loss: {:.5f}'.format(test_loss))
    print(f'Learning rate: {optimizer.param_groups[0]["lr"]:.6f}')

if __name__ == "__main__":
    for epoch in range(1, num_epochs + 1):
        train(epoch)
        test(epoch)

ERROR:

Arrhythmia_Classifier.py::test ERROR                                     [100%]
test setup failed
file C:\Users\Usama Khan\Downloads\Arrhythmia-CNN-master\Arrhythmia-CNN-master\Arrhythmia_Classifier.py, line 140
  def test(epoch):
E       fixture 'epoch' not found
>       available fixtures: anyio_backend, anyio_backend_name, anyio_backend_options, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

C:\Users\Usama Khan\Downloads\Arrhythmia-CNN-master\Arrhythmia-CNN-master\Arrhythmia_Classifier.py:140

========================= 1 warning, 1 error in 3.98s =========================
